Content.Server/Cargo/Systems/PricingSystem.cs

```diff
@@ -385,10 +384,10 @@ private double GetVendPrice(EntityPrototype prototype)
     {
         var price = 0.0;
 
-        if (prototype.Components.TryGetValue(_factory.GetComponentName(typeof(VendPriceComponent)), out var vendProto))
+        if (prototype.Components.TryGetValue(_factory.GetComponentName(typeof(StaticPriceComponent)), out var vendProto))
         {
-            var vendPrice = (VendPriceComponent) vendProto.Component;
-            price += vendPrice.Price;
+            var vendPrice = (StaticPriceComponent) vendProto.Component;
+            price += vendPrice.VendPrice;
```

Comment:
Just double checking here - in the case there's a StaticPrice with a `price` field but no `vendPrice`, the check passes, price gets its default value of 0, and it behaves as it did previously, returning 0?

Reply:
Yes